### PR TITLE
Doc for Beta feature PodSchedulingReadiness

### PR DESCRIPTION
**Pull request by Huang-Wei**
Ref https://github.com/kubernetes/enhancements/issues/3521.

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md [hunk 1]
```diff
@@ -164,7 +164,8 @@ For a reference to old feature gates that are removed, please refer to
 | `PodDisruptionConditions` | `false` | Alpha | 1.25 | 1.25 |
 | `PodDisruptionConditions` | `true` | Beta | 1.26 | |
 | `PodHasNetworkCondition` | `false` | Alpha | 1.25 | |
-| `PodSchedulingReadiness` | `false` | Alpha | 1.26 | |
+| `PodSchedulingReadiness` | `false` | Alpha | 1.26 | 1.27 |
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates.md:167]:
```suggestion
| `PodSchedulingReadiness` | `false` | Alpha | 1.26 | 1.26 |
```

**Review by sftim — COMMENTED** [review]
One nit

File: content/en/docs/reference/command-line-tools-reference/feature-gates.md (hunk 1 above)
Comment (Huang-Wei) [reply content/en/docs/reference/command-line-tools-reference/feature-gates.md]: thanks Tim. Updated.

**Comment by tengqm** [comment]
Thanks.
/lgtm
/approve

**Pull request merged**